src/components/Capture/Document.tsx

```diff
@@ -173,7 +173,7 @@ class Document extends Component<Props> {
       side,
       trackScreen,
       translate,
-      uploadFallback,
+      uploadFallback = false,
```

Comment:
So by default we want `uploadFallback` to be `false`? I believe the default for selfie is true

Reply:
That's true. I've confused this `uploadFallback` default value with `isUploadFallbackDisabled` 🙈